lib/core.js

```diff
@@ -52,6 +57,10 @@ function Zone(parentZone, data) {
 Zone.prototype = {
   constructor: Zone,
 
+  path: function () {
+    return (this.parent ? this.parent.path() : '') + '/' + this.$id;
```

Comment:
slash seems like a weird delimiter. how about ` > ` or ` -> ` (note the surrounding spaces added for readability)